pontoon/base/static/js/translate.js

```diff
@@ -320,18 +320,18 @@ var Pontoon = (function (my) {
                 '" title="Copy Into Translation (Tab)">' +
                   '<header class="clearfix' +
                     ((self.user.isTranslator) ? ' translator' :
-                      ((self.user.email === this.email && !this.approved) ?
+                      ((self.user.username === this.username && !this.approved) ?
                         ' own' : '')) +
                     '">' +
                     '<div class="info">' +
-                      ((!this.email) ? '<span title="' + self.getApproveButtonTitle(this) + '">' + this.user + '</span>' :
+                      ((!this.username) ? '<span>' + this.user + '</span>' :
                         '<a href="/contributors/' + this.username + '" title="' + self.getApproveButtonTitle(this) + '">' + this.user + '</a>') +
```

Comment:
Pontoon had some issues with XSS attacks, I think we should use something that will escape username as we do in the jinja2 templates.